--- scs-2.1.1_2.orig/src/scsmodule.c	2019-08-31 19:01:08.381691591 +0200
+++ scs-2.1.1_2/src/scsmodule.c	2019-08-31 20:45:34.401994799 +0200
@@ -297,10 +297,10 @@
 /* parse the arguments and ensure they are the correct type */
 #ifdef DLONG
 #ifdef SFLOAT
-  char *argparse_string = "(ll)O!O!O!O!O!O!|O!O!O!lffffflz(OOOOOO)";
+  char *argparse_string = "(ll)O!O!O!O!O!O!|O!O!O!LfffffLz(OOOOOO)";
   char *outarg_string = "{s:l,s:l,s:f,s:f,s:f,s:f,s:f,s:f,s:f,s:f,s:f,s:s}";
 #else
-  char *argparse_string = "(ll)O!O!O!O!O!O!|O!O!O!ldddddlz(OOOOOO)";
+  char *argparse_string = "(ll)O!O!O!O!O!O!|O!O!O!LdddddLz(OOOOOO)";
   char *outarg_string = "{s:l,s:l,s:d,s:d,s:d,s:d,s:d,s:d,s:d,s:d,s:d,s:s}";
 #endif
 #else
